src/seedu/addressbook/parser/Parser.java

```diff
@@ -72,34 +62,36 @@ public Command parseCommand(String userInput) {
         final String arguments = matcher.group("arguments");
 
```

Comment:
Switch and case should be on the same indentation.

Reply:
Yes, please follow the [Course's coding standard](https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/intermediate.html). The Google's coding style has +2 space indentation but the course's coding standard has specified it and hence "override" it. You will need to configure your IntelliJ to adhere to the style imposed in this module.